packages/vincent-tools/vincent-tool-erc20-approval/README.md

```diff
@@ -1,6 +1,47 @@
-# vincent-tool-erc20-approval
+# Vincent Tool ERC20 Approval
 
-This library was generated with [Nx](https://nx.dev).
+A tool to send ERC20 approve/allowance transactions from a Vincent app on behalf of the delegator.
+
+## Overview
+
+The Vincent Tool ERC20 Approval is part of the Vincent Tools ecosystem and is built using the Vincent Tool SDK. It allows Vincent apps to request and manage ERC20 token approvals on behalf of users, enabling seamless integration with DeFi protocols and other token-based applications.
+
+## Features
+
+- Request ERC20 token approvals for users
+- Set specific allowance amounts
+- Revoke existing approvals
+- Support for multiple ERC20 tokens
+
+## Installation
+
+```bash
+npm install @lit-protocol/vincent-tool-erc20-approval
+```
+
+## Usage
+
+This tool can be used in Vincent apps to manage ERC20 approvals:
+
+```typescript
+import { getVincentToolClient } from '@lit-protocol/vincent-sdk';
```

Comment:
:+1: Will update this when I merge the `VincentToolClient` that uses the actual vincentTool object into the `vincent-sdk` tomorrow